rct229/utils/pint_utils.py

```diff
@@ -147,7 +147,7 @@ def calcq_to_str(unit_system, obj) -> str:
     return retval
 
 
-def pint_sum(qty_list, default=None):
+def sum(qty_list, default=None):
```

Comment:
we do not need to change this function name from `pint_sum` to `sum`.
The intention was to replace the `pint_sum` with the python native `sum` function

Reply:
Good point! Addressed.